test/logging.js

```diff
@@ -1,7 +1,7 @@
 import * as log from 'loglevel'
 
 // Adjusting log level for debugging can be done here, or in specific tests that need more finegrained logging during development
-const level = 3
+const level = 1
```

Comment:
Probably want to bump this back to 3, otherwise builds might fail due to long logs. (maybe that's what happens in waardepapieren, rather than here, but it can't hurt.